app/assets/stylesheets/refinery/find_out.scss

```diff
@@ -50,7 +50,10 @@
       background: none;
     }
 
-    background-image: image-url('group-2111.png');
+    background-image: image-url('376-find-out-banner.png');
+    background-position: center center;
```

Comment:
Would this work the same way if we used a single `center`?

Reply:
You're right, a single `center` does the same as two. I remember reading that if just one orientation is listed, the second is automatically assumed to be `center`; I reckon I read that _after_ I wrote this and forgot to update.